### ISSUE TITLE
Determine if accessibility checks can be run without failing a build

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#24335]
## Issue Description
Research and define an approach where accessibility checks can be found and reported, but in a way that does not block builds. This task applies to both the Nightwatch a11y sitemap test, and the axe scans that are present in the Nightwatch and Cypress E2E

---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
